dist/pom.xml

```diff
@@ -333,9 +366,40 @@
             <dependency>io.micrometer:micrometer-core</dependency>
             <dependency>io.micrometer:micrometer-registry-prometheus</dependency>
 
+            <dependency>org.openapitools:jackson-databind-nullable</dependency>
+            <dependency>javax.annotation:javax.annotation-api</dependency>
+
           </usedDependencies>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.openapitools</groupId>
+        <artifactId>openapi-generator-maven-plugin</artifactId>
+        <version>6.2.1</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>generate</goal>
+            </goals>
+            <configuration>
+              <!-- https://github.com/OpenAPITools/openapi-generator/tree/master/modules/openapi-generator-maven-plugin -->
+              <!-- https://github.com/OpenAPITools/openapi-generator/blob/master/docs/generators/spring.md -->
+              <inputSpec>${project.basedir}/src/main/resources/api/backup-management-api.yaml</inputSpec>
+              <generatorName>spring</generatorName>
```

Comment:
`java` is for client side generation. `spring` is for server side. That said, since we are only generating the models, I think it shouldn't matter which one we use. The models should be same for client and server. I did not test with `java` though.

Reply:
[java](https://github.com/OpenAPITools/openapi-generator/blob/master/docs/generators/java.md) vs [spring](https://github.com/OpenAPITools/openapi-generator/blob/master/docs/generators/spring.md)

Reply:
I see :+1: I shortly tested with the java generator but that seems to use GSON instead of jackson so I think the spring one is preferable for use anyway.